src/System.Private.ServiceModel/tools/IISHostedWcfService/App_code/MahojongTypes.cs

```diff
@@ -0,0 +1,240 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
I'd suggest we don't purge things in the init checkin as it's time consuming to decide what are really not being used. The only certain way is to do a test run.